dispatcher_plugin_nb2workflow/dataserver_dispatcher.py

```diff
@@ -257,3 +230,17 @@ def run_query(self,
                                  extra_message = res.text)
 
         return res, query_out
+
+    def update_param_dict_download_file_url(self, param_dict):
+        for param in param_dict:
+            # TODO improve this check, is it enough?
+            if param_dict[param] != '' \
+                    and validators.url(str(param_dict[param])) \
+                    and '&token=INSERT_YOUR_TOKEN_HERE' in param_dict[param]:
```

Comment:
where is token set like that?

Reply:
> where is token set like that?

It's proposed in https://github.com/oda-hub/dispatcher-app/pull/666

Reply:
token is set on the nb2workflow backend https://github.com/oda-hub/nb2workflow/pull/171